<launch>
    <arg name="frame_id_front" default="axesFront"/>
    <arg name="frame_id_rear" default="axesRear"/>
    <arg name="frame_id_output" default="nav_base"/>
    <arg name="frame_id_map" default="map"/>

    <node name="switch_nav_base_node" pkg="switch_nav_base" type="switch_nav_base_node" output="screen">
        <param name="frame_id_front"        value="$(arg frame_id_front)"/>
        <param name="frame_id_rear"        value="$(arg frame_id_rear)"/>
        <param name="frame_id_output"        value="$(arg frame_id_output)"/>
        <param name="frame_id_map"        value="$(arg frame_id_map)"/>
    </node>
</launch>
